chirp/drivers/ft1d.py

```diff
@@ -1826,6 +1979,195 @@ def _get_scan_settings(self):
 
         return menu
 
+    def _get_digital_settings(self):
+        topmenu = RadioSettingGroup("digital_settings", "Digital")
+        menu = RadioSettingGroup("settings", "Digital Modes")
+        topmenu.append(menu)
+        GMmenu = RadioSettingGroup("first_settings", "Group Monitor(GM)")
+        topmenu.append(GMmenu)
+        WXmenu = RadioSettingGroup("WiresX_settings", "Wires-X")
+        topmenu.append(WXmenu)
+#        LOG.debug("Starting digital settings:\nMenu: '%s'"
+#                  "GMmenu: '%s'" % (topmenu, GMmenu))
```

Comment:
Can you remove these commented out lines? I appreciate you removing some old stale ones in the previous commit :)